File: 0378-kth-smallest-element-in-a-sorted-matrix/0378-kth-smallest-element-in-a-sorted-matrix.cpp
class Solution {
public:
    int kthSmallest(vector<vector<int>>& matrix, int k) {
        
        
        vector<int>ans;
        for(int i=0;i<matrix.size();i++){
            
            for(int j=0;j<matrix[0].size();j++){
                
                ans.push_back(matrix[i][j]);
                
            }
        }
        sort(ans.begin(),ans.end());
        
        if(k<=ans.size()){
            return ans[k-1];
        }
        return -1;
    }
};

File: 0378-kth-smallest-element-in-a-sorted-matrix/0378-kth-smallest-element-in-a-sorted-matrix.java
class Solution {
    public int kthSmallest(int[][] matrix, int k) {
        
        ArrayList<Integer> ans=new ArrayList<>();
        
          
        for(int i=0;i<matrix.length;i++){
            
            for(int j=0;j<matrix[0].length;j++){
                
                ans.add(matrix[i][j]);
                
            }
        }
        Collections.sort(ans);
        
        if(k<=ans.size()){
            return ans.get(k-1);
        }
        return -1;
        
    }
}